recipes/vulkan-loader/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "libvulkan-loader" %}
+{% set version = "1.3.239.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+# For this repo, v**** tags exists, but apparently the stable tags (the one corresponding to versions
+# that are available for download at https://vulkan.lunarg.com/sdk/home) are the sdk-*** ones
+# See https://github.com/KhronosGroup/Vulkan-Headers#version-tagging-scheme
+source:
+  url: https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/sdk-{{ version }}.tar.gz
+  sha256: fa2078408793b2173f174173a8784de56b6bbfbcb5fb958a07e46ef126c7eada
+
+build:
+  number: 0
+  ignore_run_exports_from:
+    - libxcb
+
+requirements:
+  build:
+    - cmake
+    - ninja
+    - {{ compiler('cxx') }}
+    - python
+  host:
+    - libvulkan-headers {{ version }}
+    - libxcb  # [linux]
+    - xorg-libx11  # [linux]
+    - xorg-libxrandr  # [linux]
+  run_constrained:
+    - libvulkan-headers {{ version }}
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libvulkan${SHLIB_EXT}  # [unix]
+    - if not exist %LIBRARY_LIB%\\vulkan-1.lib exit 1  # [win]
+    - if not exist %LIBRARY_BIN%\\vulkan-1.dll exit 1  # [win]
+
+about:
+  home: https://github.com/KhronosGroup/Vulkan-Loader
+  license: Apache-2.0
+  license_file: LICENSE.txt
+  summary: Khronos official Vulkan ICD desktop loader for Windows, Linux, and MacOS.
+
+extra:
+  feedstock-name: vulkan-loader
```

Comment:
why this distinction?

Reply:
In the past I packaged several repositories called `<something>` that just contained a C/C++ library, so I initially created a feedstock called `lib<something>-feedstock`, and then more packages were added (such as `<something>-python`, etc etc), and it turned out to be confusing for contributors to understand why the feedstock was called `lib<something>-feedstock` while the original repo was called `<something>`, and the feedstock name could not be (easily) changed. So, I started to create all the feedstosk by using the original repo name, a bit like debian source packages are named. Anyhow, I do not feel strongly about this, so I am happy to just name the feedstock as well `libvulkan-loader`.

Reply:
thank you for explaining.